packer-plugin-sdk/packer/communicator_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
-	"github.com/hashicorp/packer/packer-plugin-sdk/iochan"
+	"github.com/mitchellh/iochan"
```

Comment:
good catch !

Reply:
yeah turns out the internal iochan package was barely used anymore so I deleted it and replaced it with the external one.